analysis/tvla.py

```diff
@@ -461,9 +461,8 @@ def run_tvla(ctx: typer.Context):
                 # Make sure to re-open the CW project file as we close it during
                 # the operation to free up some memory.
                 if i_step > 0:
-                    if not OTTraceLib:
-                        project.open_project()
-                        proj_waves = project.get_waves()
+                    project.open_project()
+                    proj_waves = project.get_waves()
```

Comment:
For the OT trace database it might make sense to only get the waves you really need here, i.e., to call `get_waves()` with the specific arguments. But I might be wrong and the traces are only loaded when you actually access them. At least for the CW trace database this was the case as the waves are memory-mapped under the hood.

Reply:
Good point, I've made the change. Could you please carefully check whether my changes are OK? I really want to avoid that we have a mismatch between traces and ptx/keys, e.g., trace[i] does not belongs to plaintext[i]/key[i]

Reply:
Tested with a couple of databases and I get the same results.